<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyServerSettings</class>
 <widget class="QWidget" name="KeyServerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="keyServerListGroupBox">
       <property name="title">
        <string>Keyserver List</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="addKeyServerLabel">
            <property name="text">
             <string>URL: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="addKeyServerEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="addKeyServerPushButton">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QTableWidget" name="keyServerListTable">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>default</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Keyserver Address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Security</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Avaliable</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="tipsLabel">
          <property name="text">
           <string>Tips: Please Double-click table item to edit it.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="operationsGroupBox">
       <property name="title">
        <string>Operations</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="testKeyServerButton">
            <property name="text">
             <string>Test Listed Key Server</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDelete_Selected_Key_Server">
   <property name="text">
    <string>Delete Selected Key Server</string>
   </property>
  </action>
  <action name="actionSet_As_Default">
   <property name="text">
    <string>Set As Default</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
